pint/unit.py

```diff
@@ -1109,6 +1110,9 @@ def _convert(self, value, src, dst, inplace=False):
         if isinstance(value, Decimal):
             factor = Decimal(str(factor))
 
+        if isinstance(value, Fraction):
+            factor = Fraction(str(factor))
```

Comment:
I think that (Fraction.from_float)[https://docs.python.org/3/library/fractions.html#fractions.Fraction.from_float]) would be a better choice